x-pack/test/alerting_api_integration/security_and_spaces/group2/tests/actions/builtin_action_types/cases_webhook.ts

```diff
@@ -407,5 +407,335 @@ export default function casesWebhookTest({ getService }: FtrProviderContext) {
+                updateIncidentUrl: `${casesWebhookSimulatorURL}/rest/api/2/issue/{{{external.system.id}}}e\\\\whoathisisbad4{}\{\{`,
+              },
+              secrets,
+            });
+          simulatedActionId = body.id;
+
+          proxyServer = await getHttpProxyServer(
+            kibanaServer.resolveUrl('/'),
+            configService.get('kbnTestServer.serverArgs'),
+            () => {
+              proxyHaveBeenCalled = true;
+            }
+          );
+        });
+
+        it('should respond with bad URL error when create case URL is bad', async () => {
+          await supertest
+            .post(`/api/actions/connector/${simulatedActionId}/_execute`)
+            .set('kbn-xsrf', 'foo')
+            .send({
+              params: {
+                ...mockCasesWebhook.params,
+                subActionParams: {
+                  incident: {
+                    title: 'success',
+                    description: 'success',
+                  },
+                  comments: [],
+                },
+              },
+            })
+            .then((resp: any) => {
+              expect(resp.body).to.eql({
+                connector_id: simulatedActionId,
+                status: 'error',
+                retry: false,
+                message: 'an error occurred while running the action',
+                service_message:
+                  '[Action][Webhook - Case Management]: Unable to create case. Error: Create case URL Error: Invalid protocol. ',
+              });
+            });
+          expect(proxyHaveBeenCalled).to.equal(false);
+        });
+
+        it('should respond with bad URL error when update case URL is bad', async () => {
+          await supertest
+            .post(`/api/actions/connector/${simulatedActionId}/_execute`)
+            .set('kbn-xsrf', 'foo')
+            .send({
+              params: {
+                ...mockCasesWebhook.params,
+                subActionParams: {
+                  incident: {
+                    title: 'success',
+                    description: 'success',
+                    externalId: '12345',
+                  },
+                  comments: [],
+                },
+              },
+            })
+            .then((resp: any) => {
+              expect(resp.body).to.eql({
+                connector_id: simulatedActionId,
+                status: 'error',
+                retry: false,
+                message: 'an error occurred while running the action',
+                service_message:
+                  '[Action][Webhook - Case Management]: Unable to update case with id 12345. Error: Update case URL Error: Invalid URL. ',
+              });
+            });
+          expect(proxyHaveBeenCalled).to.equal(false);
+        });
+        after(() => {
+          if (proxyServer) {
+            proxyServer.close();
+          }
+        });
+      });
+      describe('bad comment URL', () => {
```

Comment:
Is this block of tests for bad URLs or for bad json?

Reply:
whoops will update